src/strgen/strgen_base.cpp

```diff
@@ -163,29 +163,29 @@ uint StringData::Version() const
  * Count the number of tab elements that are in use.
  * @param tab The tab to count the elements of.
  */
-uint StringData::CountInUse(uint tab) const
+size_t StringData::CountInUse(size_t tab) const
 {
-	int i;
-	for (i = TAB_SIZE; --i >= 0;) if (this->strings[(tab * TAB_SIZE) + i] != nullptr) break;
-	return i + 1;
+	size_t count = TAB_SIZE;
+	while (count > 0 && this->strings[(tab * TAB_SIZE) + count - 1] == nullptr) --count;
+	return count;
 }
```

Comment:
I'm not sure whether 'wrongly named function' is in scope, but this doesn't quite count. If just finds the last non-`nullptr` value in the tab.

Reply:
I think the name is not that bad. All these strings are written to the lng file, including missing ones.